packages/github/README.md

```diff
@@ -55,3 +55,19 @@ const newIssue = await octokit.issues.create({
   body: 'Hello Universe!'
 });
 ```
+
+## Webhook Payload Typescript Definitions
```

Comment:
nit: consider blank line separating `##` header and paragraph (typical markdown style)